### PR TITLE
Track item attributes

### PR DESCRIPTION
**Pull request by Nadrieril**
Companion PR to https://github.com/AeneasVerif/charon/pull/105.

### CODE REVIEW

File: flake.nix
Comment (sonmarcho) [inline flake.nix:7]: Don't forget to update this once the Charon PR is merged

**Review by sonmarcho — APPROVED** [review]

**Comment by sonmarcho** [comment]
Looks good to me, put aside one minor comment above. Please merge this PR *right after* the corresponding PR is merged in Charon.

File: .gitignore
Comment (Nadrieril) [reply .gitignore:75]: Because the flake.lock was both gitignored and committed into the repo. We don't want to ignore updates to it

Comment (sonmarcho) [reply .gitignore:75]: Indeed

**Pull request merged**
